.github/workflows/stacks-signer-tests.yml

```
@@ -0,0 +1,33 @@
+## Run Stacks Signer Unit Tests
+name: Tests::Stacks Signer Unit Tests
+
+on:
+  workflow_call:
+
+env:
+  BITCOIND_TEST: 0
+  RUST_BACKTRACE: full
+  SEGMENT_DOWNLOAD_TIMEOUT_MINS: 3
+  TEST_TIMEOUT: 30
+
+concurrency:
+  group: stacks-core-tests-${{ github.head_ref || github.ref || github.run_id }}
```

@obycode this is why the job was cancelled - a different PR cancelled it. this name needds to be unique
```suggestion
  group: stacks-signer-tests-${{ github.head_ref || github.ref || github.run_id }}
```

---

no worries, i figured as much! i think you commit my suggestion, it should resolve the issue brice noticed